samples/oneapi/dpc/mpi/makefile_lnx

```diff
@@ -75,6 +75,8 @@ endif
 
 MPI_PATH = $(I_MPI_ROOT)
 
+# This file uses an outdated build system that is no longer supported.
```

Comment:
should we just remove this one? not in this PR though. @Alexandr-Solovev  @homksei